test/metabase/lib/fe_util_test.cljc

```diff
@@ -44,6 +44,18 @@
                  :args vector?}
                 (lib/expression-parts query expression-clause)))))))
 
+(deftest ^:parallel literals-expression-parts-test
+  (let [query (lib/query meta/metadata-provider (meta/table-metadata :users))]
+    (testing "string literal"
+      (is (=? (lib/expression-parts query "foo") "foo"))
+      (is (=? (lib/expression-parts query "") "")))
+    (testing "number literal"
+      (is (=? (lib/expression-parts query 42) 42))
+      (is (=? (lib/expression-parts query 0) 0)))
+    (testing "boolean literal"
+      (is (=? (lib/expression-parts query true) true))
+      (is (=? (lib/expression-parts query false) false)))))
```

Comment:
Unfortunately the convention for test expectations is reversed from Jest style `expect(actual).toBe(expected)`.

`(is (=? expected actual))` please.